app/scripts/contentscript.js

```diff
@@ -60,8 +60,8 @@ async function start () {
 async function setupStreams () {
   // the transport-specific streams for communication between inpage and background
   const pageStream = new LocalMessageDuplexStream({
-    name: 'contentscript',
-    target: 'inpage',
+    name: 'metamask-contentscript',
```

Comment:
If we are going to rename these streams, we should consider leaving the old ones in as well, and using both. Otherwise this will break for anyone using their own provider.

I'm not sure how many dapps use their own provider, but I know some have in the past at least.

Reply:
I walked this back.